.gitignore

```diff
@@ -15,6 +15,9 @@
 .settings/
 bin/
 
+# VSCode
+.vscode
```

Comment:
This can be mistaken as a file instead (and treated as a file extension). Can we instead set it as `.vscode/`?